orchestration/pom.xml

```diff
@@ -54,7 +54,10 @@
       <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
       <artifactId>connectivity-apache-httpclient5</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>org.springframework.ai</groupId>
+      <artifactId>spring-ai-core</artifactId>
+    </dependency>
```

Comment:
(Major/Question)

I think we considered setting this dependency as _optional_(?) Maybe we can setup a meeting to discuss expected dependency-usage and (un)expected dependency-footprint.

Reply:
Made optional